pandas/core/indexes/datetimes.py

```diff
@@ -1743,7 +1743,33 @@ def freq(self, value):
     is_year_start = _field_accessor(
         'is_year_start',
         'is_year_start',
-        "Logical indicating if first day of year (defined by frequency)")
+        """
+        Return a boolean indicating whether the date is the first day of the
+        year.
+
+        Returns
+        -------
+        is_year_start : Series of boolean.
+
+        See Also
+        --------
+        is_year_end : Return a boolean indicating whether the date is the
```

Comment:
I'm not sure this will resolve correctly, @jorisvandenbossche do you know?

I think it should be Series.dt.is_year_start

or DatetimeIndex.is_year_start. Don't think we've decided what to do here...

Reply:
Well, I was just testing it :-) 
Turns out this works (because it is the same class, it resolves OK), but I wanted to comment that for clarity I would write it in full like `DatetimeIndex.is_year_start`, but on the other hand, not writing it full ensures it links correctly in both Series.dt and DatetimeIndex case

Reply:
Ah, in that case I'm ok with leaving it "ambiguous" since in the HTML docs you can click through and get the correct one. In the REPL you have tab completion.